File: Fibonacci.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Fibonacci {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        List<Integer> sequenciaFibonacci = new ArrayList<>();
        sequenciaFibonacci.add(0);
        sequenciaFibonacci.add(1);

        System.out.print("Digite um numero: ");
        int N = sc.nextInt();
        sc.close();

        for (int i = 2; i <= N; i++) {
            int soma = sequenciaFibonacci.get(i - 1) + sequenciaFibonacci.get(i - 2);
            sequenciaFibonacci.add(soma);
        }

        System.out.println(sequenciaFibonacci);
    }
}

File: fibonacci.js
// faça uma sequência fibonacci com um número N

function calcularFibonacci() {
    const N = parseInt(document.getElementById("numero").value);
    const sequenciaFibonacci = [0, 1];

    for (let i = 2; i <= N; i++) {
        const soma = sequenciaFibonacci[i - 1] + sequenciaFibonacci[i - 2];
        sequenciaFibonacci.push(soma);
    }

    document.getElementById("resultado").textContent = "Série Fibonacci: " + sequenciaFibonacci.join(", ");
}